applications/plugins/Flexible/deformationMapping/BaseDeformationMapping.inl

```diff
@@ -541,18 +541,22 @@ void BaseDeformationMappingT<JacobianBlockType>::apply(const core::MechanicalPar
 #ifdef _OPENMP
 #pragma omp parallel for if (this->d_parallel.getValue())
 #endif
+    std::stringstream tmp ;
     for(helper::IndexOpenMP<unsigned int>::type i=0; i<jacobian.size(); i++)
     {
         out[i]=OutCoord();
-        if (i == 0) serr << "out[0] = " << out[i] << sendl;
+        if (i == 0 && this->f_printLog.getValue())
+            tmp << "out[0] = " << out[i] << msgendl;
         for(size_t j=0; j<jacobian[i].size(); j++)
         {
             size_t index=indices[i][j];
             jacobian[i][j].addapply(out[i],in[index]);
-            if (i == 0) serr << "out["<<i<<"] + jacobian["<<i<<"]["<<j<<"].addapply(out["<<i<<"],in["<<index<<"]) = " << out[i] << sendl;
+            if (i == 0 && this->f_printLog.getValue())
+                tmp << "out["<<i<<"] + jacobian["<<i<<"]["<<j<<"].addapply(out["<<i<<"],in["<<index<<"]) = " << out[i] ;
         }
     }
     dOut.endEdit();
+    msg_info() << tmp.str();
```

Comment:
@damienmarchal Shouldn't we add the `if (this->f_printLog.getValue())` here too to avoid blank logging?

Reply:
... or simply `if(!tmp.str().empty())`

Reply:
adding if (this->f_printLog.getValue()) would not help as it is alreay done into the msg_info macro...but testing for empty string is a good idea...I recomend the onliner: 
msg_info_when(!tmp.str().empty()) << tmp.str() ;

Reply:
I will add your suggestion into the fixTestScene...during the meantime it shouldn't hurt too much :)